### PR TITLE
test(tooltip): test cases for tooltip added

### PR DESCRIPTION
**Pull request by timrombergjakobsson**
**Describe pull-request**  
Adding [unit tests to Tooltip component](https://scaniaazureservices-my.sharepoint.com/:x:/r/personal/tatyana_bumazhkina_scania_com/_layouts/15/Doc.aspx?sourcedoc=%7BCA48BF6D-92CF-4E8C-8F45-6337E2002EC4%7D&file=Unit%20test%20scenarios.xlsx&action=default&mobileredirect=true)
Fixes: [CDEP-3045](https://tegel.atlassian.net/browse/CDEP-3045)

[CDEP-3045]: https://tegel.atlassian.net/browse/CDEP-3045?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/core/src/components/tooltip/test/disabled/tooltip.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    const button = page.locator('tds-button#button-2');
+
+    // Hover over the button to show the tooltip
+    await button.hover();
+    // Assuming you can uniquely identify the tooltip that should be visible on hover
+    const tooltip = page.locator('text=Text inside Tooltip');
+
+    await expect(tooltip).toBeVisible();
+    // Make sure tooltip is visible first
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0.5 });
```
Comment (theJohnnyMe) [inline packages/core/src/components/tooltip/test/disabled/tooltip.e2e.ts:26]: 0.5 indicates 50% difference in screenshots, is the value intentionally that high?

Comment (timrombergjakobsson) [reply packages/core/src/components/tooltip/test/disabled/tooltip.e2e.ts:26]: A mistake I think!

**Review by theJohnnyMe — APPROVED** [review]
LGTM! ;)

**Review by nathalielindqvist — APPROVED** [review]
Nice! 👌

**Pull request merged**